deno_typescript/lib.rs

```diff
@@ -119,13 +122,24 @@ impl TSIsolate {
     self.isolate.execute("<anon>", source)?;
     Ok(self.state)
   }
+
+  pub fn add_custom_assets(&mut self, custom_assets: Vec<(String, PathBuf)>) {
+    let mut state = self.state.lock().unwrap();
+    for (name, path) in custom_assets {
+      state.custom_assets.insert(name, path);
+    }
+  }
 }
 
 pub fn compile_bundle(
   bundle: &Path,
   root_names: Vec<PathBuf>,
+  custom_assets: Option<Vec<(String, PathBuf)>>,
```

Comment:
Nice work. I think this will solve the situation. It would be nice to have a unit test of this in action.